src/XPurchase/VirtoCommerce.XPurchase/Services/CartProductService.cs

```diff
@@ -41,10 +41,16 @@ public async Task<IEnumerable<CartProduct>> GetCartProductsByIdsAsync(CartAggreg
             if (ids == null)
             {
                 throw new ArgumentNullException(nameof(ids));
+            }
+
+            var defaultResponseGroups = ItemResponseGroup.ItemAssets | ItemResponseGroup.ItemInfo | ItemResponseGroup.Outlines | ItemResponseGroup.Seo;
+            if (!string.IsNullOrEmpty(additionalResponseGroups))
+            {
+                defaultResponseGroups |= EnumUtility.SafeParseFlags(additionalResponseGroups, ItemResponseGroup.ItemLarge);

```

Comment:
there is no need to fallback to ItemLarge. ItemAssets as default value should be enough